src/py42/util.py

```diff
@@ -96,3 +98,29 @@ def to_list(value):
     if not isinstance(value, (list, tuple)):
         return [value]
     return value
+
+
+def parse_timestamp_to_milliseconds_precision(timestamp):
+    if isinstance(timestamp, int) or isinstance(timestamp, float):
+        return convert_timestamp_to_str(timestamp)
```

Comment:
In `convert_timestamp_to_str()`, could yo uas `(float or int)` so that it knows it can accept an `int` ?

Reply:
Changed docstring.